host/xtest/aes_perf.c

```diff
@@ -148,33 +150,23 @@ static const char *mode_str(uint32_t mode)
 static void usage(const char *progname, int keysize, int mode,
 				size_t size, int warmup, unsigned int l, unsigned int n)
 {
-	fprintf(stderr, "AES performance testing tool for OP-TEE (%s)\n\n",
-		TO_STR(VERSION));
-	fprintf(stderr, "Usage:\n");
-	fprintf(stderr, "  %s -h\n", progname);
-	fprintf(stderr, "  %s [-v] [-m mode] [-k keysize] ", progname);
-	fprintf(stderr, "[-s bufsize] [-r] [-i] [-n loops] [-l iloops] \n");
-	fprintf(stderr, "[-w warmup_time]\n");
```

Comment:
I find this short summary quite useful to get a quick overview of all the options. Why remove it?